.github/workflows/devcontainer.yml

```diff
@@ -0,0 +1,84 @@
+name: Create dev container
+
+on:
+  # run every Monday at 4:00 AM
+  schedule:
+    - cron: "0 4 * * 1"
+
+  # schedule manually
+  workflow_dispatch:
+    inputs:
+      # On workflow dispatch, `branch` is selected by default
+      # You can access it in `github.ref_name`
+
+      tag_name:
+        description: "Tag name for the container"
+        required: true
+        default: "latest"
+
+      container_repository_branch:
+        description: "Branch of the container repository"
+        required: true
+        default: "main"
+
+jobs:
+  create-container:
+    if: github.event_name != 'schedule' || github.repository_owner == 'geo-engine'
+
+    runs-on: ubuntu-24.04
+
+    env:
+      TAG_NAME: latest
+      CONTAINER_REPOSITORY_BRANCH: main
+
+    steps:
+      - name: Modify TAG_NAME if on `tag_name` is set on `workflow_dispatch`
+        if: github.event.inputs.tag_name != ''
+        run: |
+          echo "TAG_NAME=${{ github.event.inputs.tag_name }}" >> $GITHUB_ENV
+
+      - name: Modify CONTAINER_REPOSITORY_BRANCH if on `container_repository_branch` is set on `workflow_dispatch`
+        if: github.event.inputs.container_repository_branch != ''
+        run: |
+          echo "CONTAINER_REPOSITORY_BRANCH=${{ github.event.inputs.container_repository_branch }}" >> $GITHUB_ENV
+
+      - name: Checkout container files
+        uses: actions/checkout@v4
+        with:
+          repository: geo-engine/geoengine-container
+          ref: ${{ env.CONTAINER_REPOSITORY_BRANCH }}
+          ssh-key: ${{ secrets.CONTAINER_GITHUB_TOKEN }}
+          path: "container"
+
+      - name: Login to quay.io
+        run: podman login -u="geoengine+bot" -p="${{secrets.QUAY_IO_TOKEN}}" quay.io
+
+      - name: Build with podman
+        run: |
+          podman build \
+            --tag geoengine:${{env.TAG_NAME}} \
+            -f container/devcontainer/Dockerfile \
+            .
+
+      - name: Push image to quay.io
+        run: podman push geoengine:${{env.TAG_NAME}} quay.io/geoengine/geoengine:${{env.TAG_NAME}}
```

Comment:
shouldn't this be geoengine/devcontainer ?

Reply:
[35bae87](https://github.com/geo-engine/geoengine/pull/1026/commits/35bae875265258dfacb8c9103e6967c7c934406a)